source/material_model/steinberger.cc

```diff
@@ -446,6 +454,15 @@ namespace aspect
                              Patterns::Double (0.),
                              "The maximum thermal conductivity that is allowed in the "
                              "model. Larger values will be cut off.");
+          prm.declare_entry ("Viscosity averaging scheme", "harmonic",
+                             Patterns::Selection("arithmetic|harmonic|geometric|maximum composition"),
+                             "Method to average viscosities over multiple compositional fields. "
+                             "One of arithmetic, harmonic, geometric or maximum composition.");
+          prm.declare_entry ("Viscosity prefactors", "1",
```

Comment:
This is the name given also in the "latent heat" and "ascii reference profile" material models. I think we should probably keep it the same.

Reply:
I just checked latent heat, and it has a `Composition viscosity prefactor`. 

https://github.com/geodynamics/aspect/blob/8946fb99833d51183af22b1f2077326bb8e160b5/source/material_model/latent_heat.cc#L256-L258

The "composition reaction" model are has `Composition viscosity prefactor 1` and `Composition viscosity prefactor 2`. So I think it would make sense to make it consistent with that. 

The `Viscosity prefactors` in the "ascii reference profile" are actually for phase transitions, not for compositions. So keeping that name would be very confusing.